service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/gcp/ainotebook/post-startup.sh

```diff
@@ -14,7 +14,6 @@
 # https://github.com/DataBiosphere/terra-workspace-manager/tree/main/integration#Run-nightly-only-test-suite-locally
 # for instruction on how to run the test.
 
-set -o errexit
```

Comment:
Why are we removing this?

Reply:
This should not be removed.

Reply:
If we leave it the error trap won't work because the script will exit immediately. We also don't need it anymore, because the trap itself exits on completion.

Reply:
That doesn't sound right. errexit should cause faulty commands to start the exit process, including going through the ERR trap.

https://www.gnu.org/software/bash/manual/html_node/The-Set-Builtin.html

> -e
> Exit immediately... <snip>
>
> A trap on ERR, if set, is executed before the shell exits.